lms/static/sass/views/_shoppingcart.scss

```diff
@@ -56,6 +56,8 @@ $light-border: 1px solid $gray-l5;
 }
 
 .confirm-enrollment {
+  @include direction();
```

Comment:
@frrrances @marcotuts 

I just noticed that Ned fixed the issue that caused all the RTL problems so there's no need for that mixin anymore. See #8221. This seems to fix the bug that I reported and assigned to you @marcotuts. It's [UX-2261](https://openedx.atlassian.net/browse/UX-2261).

I will update sandbox instance so that you can see it still works without it.